tests/py/dynamo/converters/test_casts.py

```diff
@@ -0,0 +1,57 @@
+import torch
+import torch.nn as nn
+from harness import DispatchTestCase
+from torch.testing._internal.common_utils import run_tests
+from torch_tensorrt.dynamo.conversion import UnsupportedOperatorException
+
+
+class TestToCopyConverter(DispatchTestCase):
+    def test_to_copy_half(self):
+        class ToCopyHalf(nn.Module):
+            def forward(self, x):
+                y = x.to(dtype=torch.half)
+                return y
+
+        inputs = [torch.rand((1, 3, 10))]
+        self.run_test(
+            ToCopyHalf(),
+            inputs,
+            expected_ops={torch.ops.aten._to_copy.default},
+            precision=torch.half,
+            disable_passes=True,
+        )
```

Comment:
torch.rand() returns float32. So is casting to float16 allowed?

Reply:
Yes, I think casting to float16 is allowed in TorchScript, so I have left it enabled here